commands/applycmd.go

```diff
@@ -42,18 +40,22 @@ func (w *ApplyRunnerWrapper) Command() *cobra.Command {
 	return w.applyRunner.Command
 }
 
-// RunE runs the ResourceGroup CRD installation as a pre-step if an
-// environment variable exists. Then the wrapped ApplyRunner is
-// invoked. Returns an error if one happened. Swallows the
-// "AlreadyExists" error for CRD installation.
+// RunE runs the ResourceGroup CRD installation as a pre-step.
 func (w *ApplyRunnerWrapper) RunE(cmd *cobra.Command, args []string) error {
-	if _, exists := os.LookupEnv(resourceGroupEnv); exists {
-		klog.V(4).Infoln("wrapper applyRunner detected environment variable")
-		err := live.ApplyResourceGroupCRD(w.factory)
-		if err != nil && !apierrors.IsAlreadyExists(err) {
+	// Install ResourceGroup CRD if it does not already exist, and reset
+	// the RESTMapper to make the CRD available.
+	err := live.ApplyResourceGroupCRD(w.factory)
```

Comment:
This doesn't seem to wait for the CRD to become `Established`. Without it, the command might still report that the resource type doesn't exist: https://github.com/GoogleContainerTools/kpt/issues/1612

Reply:
Somewhat related: Given this is a long-running operation, is there a user-facing message saying kpt is installing CRD?

Reply:
@frankfarzan Currently, the CRD application is silent for first-time `preview` and `apply`, and it does not give any feedback during this operation. Should I add a message?

Reply:
I think we should think about whether we want to install the RG CRD when someone is running the `preview` command. If I do preview, I don't expect any resources to be applied.